juntagrico/util/batchmailer.py

```diff
@@ -0,0 +1,101 @@
+import copy
+from multiprocessing.connection import wait
+from django.utils import timezone
+from django.conf import settings
+import time
+import threading
+
+class Mailer:
+
+    def send(msg):
+
+        def _send_batches(msg, email_batch_size, wait_time_batch):
+            tos = msg.to
+            ccs = msg.cc
+            bccs = msg.bcc
+            num_recipients = len(bccs) + len(tos) + len(ccs)
+            recipients_batches = []
+            # use default mail strategy if num of recipients is below batch size
+            if num_recipients <= email_batch_size:
+                msg.send()
+                batch = {'timestamp': str(timezone.now()),'to': tos, 'cc': ccs, 'bcc': bccs}
+                recipients_batches.append(batch)
```

Comment:
these lines have no effect, the function returns in the next line, without using the variable.